go.mod

```diff
@@ -39,3 +39,143 @@ require (
 	go.uber.org/fx v1.14.2
 	go.uber.org/zap v1.19.0
 )
+
+require (
+	filippo.io/edwards25519 v1.0.0-alpha.2 // indirect
+	github.com/DataDog/zstd v1.4.1 // indirect
```

Comment:
I would need some help with understanding why this was not included/displayed for go 1.16 🧐